dependency-manager-tests/cocoapods/Makefile

```diff
@@ -10,5 +10,5 @@ test:
 		@echo "⚙️  Configuring CPProject with remote branch: '${GIT_REFERENCE}'..."
 		@sed "s|REMOTE_GIT_REFERENCE|${GIT_REFERENCE}|g" Podfile.src > Podfile
 		@rm -rf Pods/
-		pod update
+		pod install
```

Comment:
Why changing to `pod install`? If I have a `Podfile.lock` in my local (it's gitignored) and want to try out my new branch, `make test-cocoapods` will install the previous setup for me instead of updating to the new one.